DashAI/back/routers/experiments.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter
+
+from DashAI.back.routers.session_class import session_info
+
+router = APIRouter()
+
+# TODO To be rethinked. This is not working as intended.
+
+
+@router.post("/experiment/")
+async def run_experiment():
+    main_task = session_info.task
+    main_task.run_experiments(session_info.dataset)
+    return {"message": "Experiment done"}
+
+
+@router.get("/experiment/")
+async def get_experiment():
```

Comment:
Call with queries not implemented.
Maybe it would be good if for this module that everything raise a `NotImplementedError` or something like that.

Reply:
This module needs to be reconsidered. It has "placeholders" of how things were called before, mainly:
```
main_task.run_experiments(session_info.dataset)
```
and
```
main_task.experimentResults
```

Reply:
Ok, let's solve it in another pr.